clang/lib/Sema/SemaConcept.cpp

```diff
@@ -711,9 +711,32 @@ bool Sema::addInstantiatedCapturesToScope(
 
   unsigned Instantiated = 0;
 
+  // FIXME: This is a workaround for not having deferred lambda body
+  // instantiation.
+  // When transforming a lambda's body, if we encounter another call to an
+  // inline lambda that contains a constraint expression, we add all of the
```

Comment:
```suggestion
  // When transforming a lambda's body, if we encounter another call to a
  // nested lambda that contains a constraint expression, we add all of the
```

I think that's what you mean?